loafwallet/BundleExtension.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+var _bundle: UInt8 = 0
+
```

Comment:
I love this solution.  If I understand correctly, BundleEx does the heavy lifting of converting all the strings to the chosen language in a DQ?  Its quite smooth

Only thing is I generally prefer `guard let ` rather than forced unwrap. 

Can we do :

```
guard let bundle = Bundle(path: Bundle.main.path(forResource: temp, ofType: "lproj") else {
	 LWAnalytics.logEvent(itemName: ._20200112_ERR)
return
}
```

Maybe it will never be called but it will help us sleep a little better.

Reply:
Sure thing, looks like i missed it after I finished prototyping. On it.